doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -59,6 +59,7 @@ Thread
 
 * Added support for the Link Metrics and CSL Thread v1.2 features for the nRF53 Series devices.
 * Removed support for the :ref:`thread_architectures_designs_cp_ncp` architecture and the related tools.
+* Memory requirements page shows requirements based on the configuration used for certification instead of minimal configuration which had been removed.
```

Comment:
```suggestion
* Memory requirements page shows requirements based on the configuration used for certification instead of minimal configuration which has been removed.
```